### PR TITLE
[IN-2052] add some tags to DD MMs and template the config

### PR DESCRIPTION
**Pull request by bitrise-tom**

### CODE REVIEW

**Review by MrBlaise — APPROVED** [review]

File: inventory/ATL01-DED001 [hunk 1]
```diff
@@ -1,6 +1,11 @@
 [all:vars]
 controller=10.200.15.22
+generation=gen2
 env=dedicated
+dc=ATL01-DED001
+
+[macmini:vars]
+machine_type=mmg4
```
Comment (lmesz-bitrise) [inline inventory/ATL01-DED001:8]: Where do you use this var?

Comment (bitrise-tom) [reply inventory/ATL01-DED001:8]: added as a datadog tag

Comment (bitrise-tom) [reply inventory/ATL01-DED001:8]: or it should be :see_no_evil:

File: roles/datadog/tasks/main.yml [hunk 2]
```diff
@@ -14,14 +14,25 @@
...
+
+- name: Set DD to start at boo
+  become: True
+  shell: launchctl load -w /Library/LaunchDaemons/com.datadoghq.agent.plist
+  when: datadog_agent_status.rc != 0
+
 - name: datadog main config
-  copy:
-    src: confs/datadog.conf.yaml
+  template:
```
Comment (peterheja-bitrise) [inline roles/datadog/tasks/main.yml:34]: <3

File: roles/datadog/tasks/main.yml [hunk 3]
```diff
@@ -14,14 +14,25 @@
...
     DD_API_KEY: "{{ datadog_api_key }}"
   when: datadog_agent_status.rc != 0
   register: install
 
+- name: Setup DD launchctl
+  become: True
+  shell: cp '/opt/datadog-agent/etc/com.datadoghq.agent.plist.example' /Library/LaunchDaemons/com.datadoghq.agent.plist
+  when: datadog_agent_status.rc != 0
+
+- name: Set DD to start at boo
```
Comment (peterheja-bitrise) [inline roles/datadog/tasks/main.yml:28]: boot

File: inventory/group_vars/all.yml [hunk 4]
```diff
@@ -1,4 +1,4 @@
 ---
 ansible_connecion: ssh
 gcp_anka_bridge_registry_host: 34.123.210.118
-
+datadog_api_key: <dummy value>
```
Comment (peterheja-bitrise) [inline inventory/group_vars/all.yml:4]: I'd put apostrophes around this.

**Review by peterheja-bitrise — DISMISSED** [review]
Left some comments.

**Review by MrBlaise — APPROVED** [review]

**Review by lmesz-bitrise — APPROVED** [review]
Big like for the name of the branch :D

**Pull request merged**